src/iterate.jl

```diff
@@ -0,0 +1,138 @@
+
+export knots
```

Comment:
Please move to Interpolations.jl file so the imports and exports are all together.